staff/daniel-tomlinson/README.md

```diff
@@ -0,0 +1 @@
+My [GitHub](https://github.com/donyol/neoland-bootcamp-202207).
```

Comment:
dan, wtf! this is not what we expected from you, in day 2!!! you should setup here your github account link

Reply:
Okay bro!! Thanks Manu and toni ;D